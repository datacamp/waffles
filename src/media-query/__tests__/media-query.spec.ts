import { mediaQuery } from '../index';

describe('mediaQuery', () => {
  it('exposes media queries generated from breakpoints', () => {
    expect(mediaQuery).toMatchSnapshot();
  });
});
