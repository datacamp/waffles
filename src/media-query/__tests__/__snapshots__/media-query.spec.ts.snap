// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mediaQuery exposes media queries generated from breakpoints 1`] = `
Object {
  "large": "@media screen and (min-width: 1480px)",
  "medium": "@media screen and (min-width: 820px)",
  "small": "@media screen and (min-width: 480px)",
}
`;
